projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.html

```diff
@@ -1,5 +1,5 @@
 <!-- Have to apply styles inline because of the overlay outlet ... -->
-<igx-drop-down #inputGroupConditions [height]="'200px'" (onSelection)="onConditionsChanged($event)">
+<igx-drop-down #inputGroupConditions [displayDensity]="displayDensity" [height]="'200px'" (onSelection)="onConditionsChanged($event)">
```

Comment:
```suggestion
<igx-drop-down #inputGroupConditions [displayDensity]="column.grid.displayDensity" [height]="'200px'" (onSelection)="onConditionsChanged($event)">
```

Reply:
I didn't use column.grid.displayDensity since the inputGroup is cosy for comfortable & cosy densities and the comfortable dropdown wouldn't fit the cosy input.